### PR TITLE
View score even team has no statuses for cyphers

### LINKED ISSUE #198: Vypocet score - pad, pokud nema tym prirazeny statusy u jednotlivych sifer
ve StatusService: 
private Status getStatusByTeamAndCypher(final Team team, final Cypher cypher) {
        Status status = statusRepository.findByTeamAndCypher(team, cypher);
        if (status == null) {
            throw new IllegalStateException("No status found for the given cypher and team in database");
        }

        return status;
    }

Fungovalo by, kdyby se misto vyhozeni vyjimky vratil novy status pro tym a sifru, ktery by mel CypherStatus pending sifry? Nebo je to nekoncepcni?

### PR DESCRIPTION
**Pull request by daweedek2**
pridano overeni, ze pokud tym jeste nezahajil hru (takze tym nema prirazen status pro jednotlive sifry), 
tak je skore = 0 a organizatorska cast (game/score) je zobrazeno a aplikace nepada

Fixes #198

### CODE REVIEW

File: src/main/java/dk/cngroup/lentils/service/StatusService.java [hunk 1]
```diff
@@ -30,20 +33,24 @@ public void markCypher(final Cypher cypher, final Team team, final CypherStatus
...
+        if (getStatusByTeamAndCypher(team, cypher) == null) {
+            return 0;
+        } else {
+            return getStatusByTeamAndCypher(team, cypher).getCypherStatus().getStatusValue();
+        }
     }
 
     private Status getStatusByTeamAndCypher(final Team team, final Cypher cypher) {
         Status status = statusRepository.findByTeamAndCypher(team, cypher);
-        if (status == null) {
```
Comment (mjanousek) [inline src/main/java/dk/cngroup/lentils/service/StatusService.java]: Tohle tady nechej a misto ifu pouzij try/catch.

File: src/main/java/dk/cngroup/lentils/service/StatusService.java [hunk 2]
```diff
@@ -30,20 +33,24 @@ public void markCypher(final Cypher cypher, final Team team, final CypherStatus
     }
 
     public int getStatusScore(final Team team, final Cypher cypher) {
-        return getStatusByTeamAndCypher(team, cypher).getCypherStatus().getStatusValue();
+        if (getStatusByTeamAndCypher(team, cypher) == null) {
```
Comment (mjanousek) [inline src/main/java/dk/cngroup/lentils/service/StatusService.java]: Tady pouzij try catch misto ifu ;)

File: src/main/java/dk/cngroup/lentils/service/StatusService.java (hunk 1 above)
Comment (daweedek2) [reply src/main/java/dk/cngroup/lentils/service/StatusService.java]: pokud pouziju try / catch tak to spadne - nullpointer exception

Comment (daweedek2) [reply src/main/java/dk/cngroup/lentils/service/StatusService.java]: hotovo

File: src/main/java/dk/cngroup/lentils/service/StatusService.java (hunk 2 above)
Comment (daweedek2) [reply src/main/java/dk/cngroup/lentils/service/StatusService.java]: upraveno

**Pull request merged**